### ISSUE TITLE
Icons next to line numbers are missing

### ISSUE THREAD

**Issue by imnr** [facelessuser/BracketHighlighter#536]
I've been using **BracketHighlighter** for a long time and everything was fine but since yesterday the Icons next to line numbers have been missing. There's also no highlights available for any kind of brackets.
I've tried reinstalling BracketHighlighter using package control but the problem is not solved. I did also reinstalled sublime text and it didn't solve the problem either. 

I'm using BracketHighlighter alongside SublimeLinter and SublimeREP
My python version is 3.7.4.
The console doesn't show any related logs about this issue.

I guess the problem is about **backrefs ver** and **mdpopups ver** which are not available via support info
Support Info:
- ST ver.: 3207
- Platform: windows
- Arch: x64
- Plugin ver.: 2.27.7
- Install via PC: True
- mdpopups ver.: Version could not be acquired!
- backrefs ver.: Version could not be acquired!
- markdown ver.: 2.6.11
- pygments ver.: 2.1a0
- jinja2 ver.: 2.10.1

I would appreciate your help.

**Comment by facelessuser** [comment]
1. I would go to your Packages folder and delete the backrefs folder and mdpopups folder (if they are there)
2.  Run `Package Control: Satisfy dependencies`.
3. Watch Sublime console for completion, and potentially any errors in install
4. If all is successful, restart Sublime
5. Open the console and look for errors.

**Comment by imnr** [comment]
Thank you, It worked. 
I also realized that every time I reinstalled BracketHighlighter, my package control stopped working. So  I had to do the following steps:

1. Opening setting from `Preferences >Settings`.
2. From the user section looking for `ignored packages`
3. Deleting the line containing `0_package_control_loader`
4. Deleting backrefs folder and mdpopups folder from Pacakges folder
5. Running `Package Control: Satisfy dependencies`
6. Restarting Sublime
